CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+-  Error for Teku and Lighthouse validator clients on Windows.
```

Comment:
If you were a user, how helpful would the current message be for you to know if the new version fixes a problem you face or might face? Moreover, when we later check the Changelog, we won't have a clue what we fixed here because of the lack of details.

```suggestion
-  Teku and Lighthouse import keys container error on Windows.
```